build.gradle

```diff
@@ -46,7 +46,7 @@ subprojects {
     targetCompatibility = '1.8'
 
     ext {
-        jetVersion = '3.1'
+        jetVersion = '3.2'
```

Comment:
I am assuming this is fine as soon as 3.2 is released.

Reply:
👍